corehq/apps/domain/decorators.py

```diff
@@ -80,51 +80,52 @@ def login_and_domain_required(view_func):
     def _inner(req, domain, *args, **kwargs):
         user = req.user
         domain_name, domain = load_domain(req, domain)
-        if domain:
```

Comment:
view this commit with https://github.com/dimagi/commcare-hq/pull/19289/commits/d3db8ca7ec6cdeef0e28ff9442869a4551690e99?w=1